### PR TITLE
Added Lan button into the Options dialog

### LINKED ISSUE #6601: No Lan button in the Tools - Options dialog
Tools Options
Hello @N-thony, 

Please could you add a lan button to the Tools - Options dialog? 

Thanks, 

Rachel

### PR DESCRIPTION
**Pull request by N-thony**
Fixes #6601
@africanmathsinitiative/developers @rdstern @rachelkg this is ready for review. Thanks.

### CODE REVIEW

**Comment by lloyddewit** [comment]
@HawardKetoyoMsatsi please could you review?
@rdstern Please could you test?
Thanks

**Comment by rdstern** [comment]
This seems to work ok and could approve.  Except it doesn't work when you press the Apply button.

image

So here it is with Lan active, but in English.

And here I am in French without the Lan

image

When I go out and then in to the dialogue again, then it is ok.  But if the options are easy to activate on Apply as well as when the dialogue is loaded, that would be more complete.

Not worth too much time though.

**Comment by N-thony** [comment]
> This seems to work ok and could approve. Except it doesn't work when you press the Apply button.
> 
> image
> 
> So here it is with Lan active, but in English.
> 
> And here I am in French without the Lan
> 
> image
> 
> When I go out and then in to the dialogue again, then it is ok. But if the options are easy to activate on Apply as well as when the dialogue is loaded, that would be more complete.
> 
> Not worth too much time though.

@rdstern thanks for your comment, I think it is now fixed/resolved. Please, have a look.

**Review by rdstern — APPROVED** [review]

**Comment by shadrackkibet** [comment]
@HawardKetoyoMsatsi your peer review is still pending here.

File: instat/dlgOptions.vb [hunk 1]
```diff
@@ -358,6 +361,21 @@ Public Class dlgOptions
...
+        Else
+            strCurrLang = frmMain.clsInstatOptions.strLanguageCultureCode
+        End If
+
+        Dim strConfiguredLanguage As String = frmMain.clsInstatOptions.strLanguageCultureCode
+        frmMain.clsInstatOptions.strLanguageCultureCode = strCurrLang
+        autoTranslate(Me)
+        SetView()
+        frmMain.clsInstatOptions.strLanguageCultureCode = strConfiguredLanguage
+
```
Comment (HawardKetoyoMsatsi) [inline instat/dlgOptions.vb:376]: Remove this empty line

**Review by HawardKetoyoMsatsi — CHANGES_REQUESTED** [review]

**Comment by HawardKetoyoMsatsi** [comment]
@shadrackkibet over to you.

**Comment by rdstern** [comment]
Excellent idea.  Perhaps @Patowhiz can add that during the sprint?

**Review by rdstern — APPROVED** [review]

**Review by shadrackkibet — APPROVED** [review]

**Pull request merged**